daphne_worker_test/tests/test_runner.rs

```diff
@@ -236,6 +236,36 @@ impl TestRunner {
             res.status, res.error
         );
 
+        let mk_config = || {
+            HpkeReceiverConfig::gen(random(), HpkeKemId::X25519HkdfSha256)
+                .expect("failed to generate receiver config")
+        };
+        let res: InternalTestAddTaskResult = t
+            .helper_post_internal(
+                &format!("{version}/internal/test/add_hpke_config"),
+                &mk_config(),
+            )
+            .await;
+
+        assert_eq!(
+            res.status, "success",
+            "response status: {}, error {:?}",
+            res.status, res.error
+        );
+
+        let res: InternalTestAddTaskResult = t
```

Comment:
We probably should replace `InternalTestAddTaskResult` with a more general name. How about `InternalTestCommandResult`?